researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -480,19 +481,22 @@ public SharingScope getUserSharingScope() {
     @Override
     public void setUserSharingScope(Context context, String scope) {
         StudyParticipant participant = new StudyParticipant();
+        SharingScope sharingScope = RestUtils.GSON.fromJson(scope, SharingScope.class);
+        participant.setSharingScope(sharingScope);
+
         participant.setSharingScope(SharingScope.valueOf(scope));
```

Comment:
Yes it is, good find.  I must have been debugging something before I pushed the code.